pytest_client_tools/plugin.py

```diff
@@ -91,15 +92,32 @@ def candlepin(request):
             "--candlepin-container-is-running"
         )
         initial_wait = 0
+        verify = False
         if start_container:
-            stack.enter_context(_create_candlepin_container())
+            container = _create_candlepin_container()
+            stack.enter_context(container)
```

Comment:
minor improvement: `ExitStack.enter_context()` returns the same value that is being pushed on it, so you can write:
```suggestion
            container = stack.enter_context(_create_candlepin_container())
```

Reply:
done